build.gradle

```diff
@@ -4,11 +4,11 @@ plugins {
 }
 
 group = "org.jenkins-ci.plugins"
-description = "Jucies update center plugin"
+description = "Plugin adds additional Jucies update center that provides additional plugins distributed in more flexible way."
```

Comment:
You can supersede. The idea that in update center plugin should have capitalised name and description.